### ISSUE TITLE
Extend validator to check for parameter selected for hierarchical optimization

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/PEtab#44]
- [ ] Check that data points sharing scaling and offset have the same noise parameter
- [ ] Check that selected parameters are actually offset/proportionality/sigma parameters
...

**Comment by dweindl** [comment]
On second thought: Not sure if that is something to implement in PEtab, since it completely depends on the implementation of hierarchical optimization. Maybe that is rather a pyPESTO- and parPE-specific extension.

**Comment by yannikschaelte** [comment]
yes, that would indeed be specific to tools supporting to do this kind of thing, so let' move the issue there?
